### DISCUSSION TITLE
InnovateAsterisk browser-Phone with freepbx

### DISCUSSION THREAD

**Discussion by jacknahid** [InnovateAsterisk/Browser-Phone#454] [Q&A]
I want to use InnovateAsterisk browser phone  my FreePBX 15. 
I have configured a self-signed SSL certificate and enabled WebRTC in the extension. 
However, the phone is not registering.

kindly have a look on attachments[](url)
![Screenshot 2023-05-09 at 02-21-05 Browser Phone](https://user-images.githubusercontent.com/42267711/236927117-2470ab48-1b5d-4708-962a-9a8c393b91aa.png)
![extension_setting](https://user-images.githubusercontent.com/42267711/236927124-8a7d5fdd-c9c2-486d-bd02-a1aaa3402108.png)
![asterisk sip setting](https://user-images.githubusercontent.com/42267711/236927134-50503ca9-e004-4679-9732-f7e964ad5ca9.png)
![freepbxadvancedsetting](https://user-images.githubusercontent.com/42267711/236927140-cb27783a-cf1e-4898-b98e-1f6e3336218e.png)

**Comment by InnovateAsterisk** [comment]
What error are you getting int he browser console output.?

**Comment by InnovateAsterisk** [comment]
Your HTTP bind address (In HTTPS conf) is set to 127.0.0.1, this address is not reachable via the web. You need this to be probably 0.0.0.0 - also make sure you have the port (8089) open on the firewall.
